module_1/src/com/epam/cycles/CyclesTaskA.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.cycles;
+import java.util.Scanner;
+
+public class CyclesTaskA {
+
+    public static void main(String[] args) {
+
+    Scanner console = new Scanner(System.in);
+    System.out.print("Enter the natural number A (non-negative integer greater than or equal to 1): ");
+    int a = console.nextInt();
+
+    if (a >= 1) {
+        int sum = 0;
```

Comment:
I see here still integer variables
```suggestion
    long a = console.nextInt();

    if (a >= 1) {
        long sum = 0;
```

Reply:
yes, I'm sorry, I missed this suggestion